package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "svg2react-icon",
-  "version": "3.0.1",
+  "version": "4.0.1",
```

Comment:
Bumping major version, since it's a breaking change for a certain version of `@types/react`. Should we?